BREAKING.md

```diff
@@ -59,6 +61,12 @@ The following deprecated methods are  now removed from sequence and merge-tree.
 ### Remove ConnectionState.Connecting
 `ConnectionState.Connecting` has been removed. Migrate all usage to `ConnectionState.CatchingUp` instead.
 
+### Remove `IContainerRuntimeBase.setFlushMode`
+The `setFlushMode` has been removed from `IContainerRuntimeBase`. Please remove all usage of this method as FlushMode is now an immutable property for the container runtime, optionally provided at creation time via the `IContainerRuntimeOptions` interface. See [#9480](https://github.com/microsoft/FluidFramework/issues/9480#issuecomment-1084790977).
+
+### Remove flush mode tracking and validation in `PendingStateManager`
```

Comment:
This is internal detail; I do not think we should put it here - it has more chances of confusing people (who have never heard of this class and may start to wonder if they need to do anything about it)
We should double check that this class is not exported from package.